src/utils/constants.ts

```diff
@@ -8,7 +8,7 @@ export const TRADITIONAL_OCL_URL =
   window.TRADITIONAL_OCL_HOST || "https://qa.openconceptlab.org";
 export const OCL_SIGNUP_URL =
   // @ts-ignore OCL_SIGNUP_URL is injected at runtime via index.html
-  window.OCL_SIGNUP_URL || "https://qa.openconceptlab.org/accounts/signup/";
+  window.OCL_SIGNUP_URL || "https://qa.openconceptlab.org/#/accounts/signup/";
```

Comment:
i find the changed link working perfect cc @hadijahkyampeire

Reply:
Thanks @HerbertYiga for testing this I will test it and approve